src/Presenters/ApiPresenter.php

```diff
@@ -78,6 +78,18 @@ public function renderDefault(): void
 
         try {
             $response = $handler->handle($params);
+            $outputValid = count($handler->outputs()) === 0; // back compatibility for handlers with no outputs defined
+            $outputValidatorErrors = [];
+            foreach ($handler->outputs() as $output) {
+                if ($output->validate($response)) {
+                    $outputValid = true;
+                    break;
+                }
+                $outputValidatorErrors[] = $output->getErrors();
+            }
+            if (!$outputValid) {
+                $response = new JsonApiResponse(500, ['status' => 'error', 'message' => 'Internal server error', 'detail' => $outputValidatorErrors]);
```

Comment:
dal by som asi ze `details` ked to bude array

Reply:
ok, to mozem, len som drzal konvenciu v inej casti API presentera :)